website/docs/d/nsxt_edgegateway.html.markdown

```diff
@@ -57,6 +57,10 @@ The following arguments are supported:
 definition at provider level.
 
 * `name` - (Required) NSX-T Edge Gateway name.
+* `read_limit_unused_ip_count` - (Optional, *v3.13+*) Sets a limit of IPs to count for
+  `used_ip_count` and `unused_ip_count` attributes to avoid exhausting compute resource while
+  counting IPs in large IPv6 subnets. Defaults to `1000000`, update is a no-op, but will affect
+  newly read data.
```

Comment:
As there's no update here
```suggestion
  counting IPs in large IPv6 subnets. Defaults to `1000000`
```

Reply:
Probably worth mentioning what a value of "0" will do, as in many cases we use it to signify "unlimited"

Reply:
Good point!
0 is `unlimited` as per SDK code https://github.com/vmware/go-vcloud-director/pull/682/files#diff-11389a99316769bddb45effb145f62650d74c0f8711c34ef9b22eb719d0d7ca5R976-R979

Updated docs.